File: src/main/java/scrumweb/dto/fieldcontent/RadioButtonContainerContentDto.java
package scrumweb.dto.fieldcontent;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import scrumweb.dto.projectfield.RadioButtonDto;

@NoArgsConstructor
@Getter @Setter
public class RadioButtonContainerContentDto extends FieldContentDto {
    private RadioButtonDto element;

    public RadioButtonContainerContentDto(Long projectFieldId, String projectFieldName, RadioButtonDto element) {
        super(projectFieldId, projectFieldName);
        this.element = element;
    }
}

File: src/frontend/src/app/model/fields-content/RadioButtonContainerContentDto.ts
import { RadioButtonDto } from './RadioButtonDto';
import { FieldContentDto } from './FieldContentDto';

export class RadioButtonContainerContentDto extends FieldContentDto {
    element: RadioButtonDto;

    constructor(projectFieldId: number, projectFieldName: string, element: RadioButtonDto) {
        super(projectFieldId, projectFieldName);
        this.element = element;
    }
}
